src/WebJobs.Script/OutOfProc/Http/Configuration/HttpInvokerOptionsSetup.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.OutOfProc.Http
+{
+    internal class HttpInvokerOptionsSetup : IConfigureOptions<HttpInvokerOptions>
+    {
+        private IConfiguration _configuration;
+        private ILogger _logger;
+
+        public HttpInvokerOptionsSetup(IConfiguration configuration, ILoggerFactory loggerFactory)
+        {
+            _configuration = configuration;
+            _logger = loggerFactory.CreateLogger<HttpInvokerOptionsSetup>();
+        }
+
+        public void Configure(HttpInvokerOptions options)
+        {
+            IConfigurationSection jobHostSection = _configuration.GetSection(ConfigurationSectionNames.JobHost);
+            var httpInvokerSection = jobHostSection.GetSection(ConfigurationSectionNames.HttpInvoker);
+            if (httpInvokerSection.Exists())
+            {
+                httpInvokerSection.Bind(options);
+                HttpInvokerDescription httpInvokerDescription = options.Description;
+
+                if (httpInvokerDescription == null)
+                {
+                    throw new HostConfigurationException($"Invalid WorkerDescription for HttpInvoker");
+                }
+                httpInvokerDescription.FixAndValidate();
+                if (string.IsNullOrEmpty(httpInvokerDescription.DefaultWorkerPath))
+                {
+                    if (!Path.IsPathRooted(httpInvokerDescription.DefaultExecutablePath))
```

Comment:
why "DefaultExecutablePath" and not just "ExecutablePath"?

Reply:
`DefaultExecutablePath` is used be existing language worker configuration. Here is a sample worker.config.json : https://github.com/Azure/azure-functions-java-worker/blob/dev/worker.config.json

Reused existing schema